ietf/context_processors.py

```diff
@@ -51,14 +39,11 @@ def social_menu(site):
 
 def global_pages(request):
     site = Site.find_for_request(request)
+    # XXX Return lazy values. This makes a big difference when a page renders
```

Comment:
What does XXX stand for?

Reply:
It's a convention meaning "something unexpected is going on here and this is why".